### PR TITLE
Add store feature

### PR DESCRIPTION
**Pull request by zkdan**

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -1,84 +1,109 @@
...
-    {
-      title: 'HackerYou Hoodie',
-      price: 50.00,
-      stock: 4,
-    }, 
-  ]
+ var totalInventory = [
+  {
+    title: 'Bowie Tee',
+    url: 'images/bowie.jpg',
```
Comment (MsMaryCodes) [inline scripts.js:6]: update image url, it's not working

**Review by MsMaryCodes — COMMENTED** [review]
great work so far, just a few changes to make and it's all good

File: scripts.js [hunk 2]
```diff
@@ -1,84 +1,109 @@
...
+  },
+  {
+    title: 'HackerYou Hoodie',
+    price: 50.00,
+    stock: 4,
+  },
+]
 
-  // document ready function allows page to load before running any of our scripts 
-$(function(){
```
Comment (jonathanphill) [inline scripts.js:48]: why does this function starts with a '$' sign ?

File: scripts.js [hunk 3]
```diff
@@ -106,6 +106,23 @@ $(function () {
...
   // STEP FOUR: add an event listener that will notice when a user clicks on a currency button, find out which currency they have selected, and call our display items method again 
 
+  $('button').on('click', function () {
+    // the ids on our buttons match properties on our currencies object *on purpose* 
+    const currency = $(this).attr('id');
+
+    // we don't want to keep adding to our list here, we want to remove the existing markup 
+    //and replace it with new stuff
+    $('.inventory').html('');
+    displayItems(currentStock, currencies[currency]);
```
Comment (GHJBaek) [inline scripts.js]: Please double cheque the currency

Comment (zkdan) [reply scripts.js]: ok!

**Pull request merged**